index.src.html

```diff
@@ -958,6 +960,103 @@ <h3 id="strip-url">
     </li>
   </ol>
 
+  <h3 id="ancestor-origin">
+    Determine the Ancestor Origin availble to a Location
+  </h3>
+  Given a <a>Location</a> <var>location</var> and a <a>browsing context</a> <var>context</var>
+  we can determine what origin information for <var>context</var> should be included in
+  <var>location</var>'s <a>ancestor origins array</a>.
+
+  <ol>
+    <li>Let <var>document</var> be <var>context</var>'s <a>active document</a>.</li>
+    <li>Let <var>documentOrigin</var> be <var>document</var>'s <a>origin</a>.</li>
+    <li>If <var>documentOrigin</var> is an <a>opaque origin</a>, return <code>"null"</code>.
+    <li>Let <var>locationOrigin</var> be <var>location</var>'s relevant <a>Document</a>'s <a>origin</a>.
+    <li>Let <var>policy</var> be <var>document</var>'s <a>referrer policy</a>.</li>
+    <li>
+      Execute the statements corresponding to the value of <var>policy</var>:
+
+      <dl class="switch">
+        <dt><a>"<code>no-referrer</code>"</a></dt>
+        <dd>Return <code>"null"</code></dd>
+
+        <dt><a>"<code>origin</code>"</a></dt>
+        <dd>Return <var>documentOrigin</var>.</dd>
+
+        <dt><a>"<code>unsafe-url</code>"</a></dt>
+        <dd>Return <var>documentOrigin</var>.</dd>
+
+        <dt><a>"<code>strict-origin</code>"</a></dt>
+        <dd>
+          <ol>
+            <li>
+              If <var>context</var> is <a>TLS-protected</a> <em>and</em>
+              <var>locationOrigin</var> is not an <a><em>a priori</em> authenticated
```

Comment:
Good point.  That's copied from section 8.3 earlier in the document, will need to fix there, too.  https://github.com/w3c/webappsec-referrer-policy/pull/78